app/api/server/v1/users.js

```diff
@@ -229,12 +230,34 @@ API.v1.addRoute('users.list', { authRequired: true }, {
 		const { offset, count } = this.getPaginationItems();
 		const { sort, fields, query } = this.parseJsonQuery();
 
-		const users = Users.find(query, {
-			sort: sort || { username: 1 },
-			skip: offset,
-			limit: count,
-			fields,
-		}).fetch();
+		const inclusiveFields = getInclusiveFields(fields);
+
+		const actualSort = sort && sort.name ? { nameInsensitive: sort.name, ...sort } : sort || { username: 1 };
+
+		const users = Promise.await(UsersRaw.col
```

Comment:
Shouldn't this return the total amount of items? (for pagination)

Reply:
Thanks!!!!
I have fixed that now. Please do have a look again :angel:

Reply:
Nice! Let me see